p22/src/main.rs

```diff
@@ -0,0 +1,130 @@
+use p22::calc::*;
+use p22::figures::*;
+use p22::tictac::*;
+
+pub mod song;
+
+fn main() {
+    println!("\n\nCalculator\n");
```

Comment:
After you finished development of a library crate, it's better to remove `main.rs` from it.